citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/service/parse/wos/tagged/CreatorTagHandler.java

```diff
@@ -31,6 +31,8 @@ protected void addFullnameToContributor(String nameString, int fieldIdx, List<Co
         // we assume that the order of standardized names is the same as of fullnames
         if (contributors.size() > fieldIdx) {
             Contributor c = contributors.get(fieldIdx);
+            c.setGivenName(name.givenName);
+            c.setSurname(name.surname);
```

Comment:
no that would override the short version of the name. I think the full name is simply not added to the extra field when submitted to zotero.